src/content/platform-integration/ios/ios-latest.md

```diff
@@ -5,6 +5,18 @@ description: >-
   the latest releases of iOS.
 ---
 
+:::warning
+Flutter's debug mode is broken on physical devices running iOS 18.4 beta.
+See [flutter#163984][] for details.
+
+Flutter recommends you do not upgrade your devices to iOS 18.4 beta.
+
+To target iOS 18.4 beta physical devices, use
+[Flutter's release or profile build modes][].
+
+Alternatively, target an iOS simulator or iOS 18.3 or lower physical device.
+:::
+
```

Comment:
WDYT of:

```
An upcoming change to iOS has caused a temporary break in Flutter's ability to debug code on physical devices running iOS 18.4 (currently in beta). See [flutter#163984][] for details.

In the meantime, we recommend these workarounds:

* When developing with a physical device, use one running iOS 18.3 or earlier.
* Use a simulator for development rather than a physical device.
* If you must use a device updated to iOS 18.4, use [Flutter's release or profile build modes][] to do so.
```

Reply:
Updated with some minor tweaks!